File: WebRoot/app/view/master/category/model/CategoryAuthority.js
Ext.define('erp.view.master.category.model.CategoryAuthority', {
	extend: 'erp.common.basic.model.Model',
	idProperty: 'czy_gh',
	fields: [
		{ name: 'lbbh' ,hidden:true},
		{ name: 'czy_gh' ,header:'操作员工号',columnWidth:80},
		{ name: 'czy_xm' ,header:'操作员姓名',columnWidth:80},
		{ name: 'tpbj', type: 'int' ,header:'特批',columnWidth:40,isSign:true}
	]
});

File: src/erp/erp/master/category/model/CategoryAuthority.java
package erp.erp.master.category.model;

import java.io.Serializable;

import erp.common.Model;

public class CategoryAuthority extends Model implements Serializable{
	/*功能：采购类别权限表
	 *作者：伍恰
	 *时间：2016/01/11
	 *数据库名：cglb_qxb
	 * */
	private static final long serialVersionUID = 1L;
	private String			lbbh;
	private String			czy_gh;
	private String			czy_xm;
	private int			tpbj;

	public String getLbbh() {
		return lbbh;
	}
	public void setLbbh(String lbbh) {
		this.lbbh = lbbh;
	}
	public String getCzy_gh() {
		return czy_gh;
	}
	public void setCzy_gh(String czy_gh) {
		this.czy_gh = czy_gh;
	}
	public String getCzy_xm() {
		return czy_xm;
	}
	public void setCzy_xm(String czy_xm) {
		this.czy_xm = czy_xm;
	}
	public int getTpbj() {
		return tpbj;
	}
	public void setTpbj(int tpbj) {
		this.tpbj = tpbj;
	}
}
